python/tests/test_data_consumers.py

```diff
@@ -25,9 +25,10 @@ def test_logger():
     logger = Logger("test_device", tempdir)
     logger.start()
     logger.put("Hello world")
+    currentTime=str(datetime.datetime.now())
     time.sleep(1.5) # Allow for some time for the data to be processed by the queue processor
     logger.stop()
 
     with open(tempdir + "/test_device-log.txt", "r") as logfile:
         log = logfile.read()
-        assert log == "Hello world\n"
+        assert log.find("Hello world\n") != -1
```

Comment:
A cleaner way to write this is `assert "Hello world" in log` but this is fine too